packages/react-dev-utils/formatWebpackMessages.js

```diff
@@ -16,7 +16,13 @@ function isLikelyASyntaxError(message) {
 
 // Cleans up webpack error messages.
 function formatMessage(message) {
-  let lines = message.split('\n');
+  let lines = [];
+
+  if (typeof lines === 'string' || lines instanceof String) {
+    lines = message.split('\n');
```

Comment:
unreachable line!

Reply:
Closed this in favor of https://github.com/facebook/create-react-app/pull/9943